src/databricks/labs/ucx/install.py

```diff
@@ -288,6 +288,15 @@ def warehouse_type(_):
         else:
             groups_config_args["auto"] = True
 
+        # Checking for override clusters
+        if os.environ.get("DATABRICKS_CLUSTER_ID", None) and os.environ.get(
```

Comment:
Ask for clusters via self._question, not the env vars

Reply:
That's the guidance I needed.